tt_metal/impl/dispatch/kernels/cq_dispatch.cpp

```diff
@@ -106,7 +107,8 @@ void completion_queue_reserve_back(uint32_t num_pages) {
 
 FORCE_INLINE
 void notify_host_of_completion_queue_write_pointer() {
-    uint64_t pcie_address = get_noc_addr_helper(pcie_noc_xy_encoding, HOST_CQ_COMPLETION_WRITE_PTR);  // For now, we are writing to host hugepages at offset
+    uint64_t completion_queue_write_ptr_addr = command_queue_base_addr + HOST_CQ_COMPLETION_WRITE_PTR;
```

Comment:
is this the same as `completion_queue_base_addr`? if thats the case then we wouldn't need that extra compile time arg

Reply:
Maybe it's named bad, this is the write ptr in the reserved region, not actually in the completion q region

Reply:
oh this is the address where we write the completion queue addr?

Reply:
Yes that's right, a bug here is that it was being hard-coded - need to pass it in for multi cq